changelog.md

```diff
@@ -1,3 +1,44 @@
+# 1.2
+
+## Features
+* Added multi objective via Par-EGO.
+
+## Major Changes
+* Results are instantly saved by default now.
```

Comment:
You may want to clarify this by writing that the result history is saved in every iteration.